.gitattributes

```diff
@@ -0,0 +1,63 @@
+###############################################################################
```

Comment:
could we move this and gitignore to the main dir in the repo? git should be able to lookup this by traversing tree up.